Branch-SDK/Branch-SDK/BNCServerInterface.m

```diff
@@ -138,10 +138,9 @@ - (void)genericHTTPRequest:(NSURLRequest *)request retryNumber:(NSInteger)retryN
             }
             
         }
-        dispatch_async(dispatch_get_main_queue(), ^{
-            if (callback)
-                callback(serverResponse, error);
-        });
+		//	Don't call on the main queue since it might be blocked.
+		if (callback)
```

Comment:
No it's not that. See line 140? It's got 8 spaces whereas these two lines have 2 tabs. It's just a weird indentation thing for these 2 lines alone.